go/api/server.go

```diff
@@ -116,18 +120,29 @@ func setRoutes() {
 		TripRepository: tripRepository,
 	}
 
+	featureHandler := handlers.FeatureHandler{
+		Repository: featureRepository,
+	}
+
 	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
 	router.Static("/backoffice", "./backoffice")
 
 	router.POST("/login", authHandler.Login)
 
 	adminsRoutes := router.Group("/admin", middlewares.UserIsLogged, middlewares.UserIsAdmin)
+
 	adminsTripRoutes := adminsRoutes.Group("/trips")
 	adminsTripRoutes.GET("", tripHandler.GetAll)
 	adminsTripRoutes.GET("/:id", tripHandler.Get)
 	adminsTripRoutes.DELETE("/:id", tripHandler.Delete)
 
+	adminsFeatureRoutes := adminsRoutes.Group("/app-settings")
+	adminsFeatureRoutes.POST("", featureHandler.Create)
```

Comment:
Pareil pour le delete

```suggestion
```

Je pense qu'il faut créer les feature en of par défault si elle existe pas en base au démarrage de l'app. Comme pour l'utilisateur admin

Comment ca on a juste la route de get et de patch. Ce qui fait plus sens, on va jamais supprimer un feature